### DISCUSSION TITLE
RX 6600 XT Suggested Settings

### DISCUSSION THREAD

**Discussion by VeteranXT** [lshqqytiger/stable-diffusion-webui-directml#225] [Q&A]
Any Suggestion
My specs is :
 Ryzen 5 5600
32 GB DDR5
RX 6600 XT 8GB

256 GB SSD 2x
1TB HDD
Window 11

Any suggestion for this machine?

**Comment by Williecm22** [comment]
Hi, been running SD webui-directml for a couple of months, these are my settings for my Radeon RX 6600. I also have:
icore i7 3rd gen, 16 GB DDR3 windows 11pro. (had this pc for 10 years, its old but works). 
I get like 1.5 it/s, and sometimes better, depending on the model i am using, and if its prune or not,

git config --global --add safe.directory H:/SD/stable-diffusion-webui-directml
set PYTHON=
set GIT=
set VENV_DIR=
set COMMANDLINE_ARGS= --opt-split-attention --medvram --disable-nan-check --always-batch-cond-uncond --upcast-sampling --autolaunch 

set SAFETENSORS_FAST_GPU=1
git pull
call webui.bat
